src/plugins/vis_type_tagcloud/public/components/tag_cloud.test.js

```diff
@@ -32,10 +32,13 @@ import _ from 'lodash';
 import d3 from 'd3';
 import 'jest-canvas-mock';
 
-import { fromNode, delay } from 'bluebird';
 import { TagCloud } from './tag_cloud';
 import { setHTMLElementOffset, setSVGElementGetBBox } from '../../../../test_utils/public';
 
+function delay(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
Are there any benefit in using `setImmediate()` over `setTimeout()`here?

Reply:
I don't think there will be any difference in performance in this case.
` setTimeout()` has the same priority as delay from `bluebird`.
`setImmediate()` can be executed earlier, which can cause errors

Reply:
As @AMoo-Miki suggested in ur other PR, this `delay` is used many times. We should move to a single place to avoid the redundance.